apps/src/amazonFutureEngineerEligibility/amazonFutureEngineerEligibility.js

```diff
@@ -0,0 +1,22 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import AmazonFutureEngineerEligibility from '@cdo/apps/templates/amazonFutureEngineerEligibility';
+
+$(document).ready(init);
+
+function showAmazonFutureEngineerEligibility() {
+  const amazonFutureEngineerEligibilityElement = $(
+    '.amazon-future-engineer-eligibility-container'
+  );
+
+  ReactDOM.render(
+    // Need to update API endpoint and source page ID (not sure if even needed)
+    <AmazonFutureEngineerEligibility />,
+    amazonFutureEngineerEligibilityElement[0]
```

Comment:
is there ever a risk of `amazonFutureEngineerEligibilityElement` being null/not an array with an element?

Reply:
For now I don't think so, but I added a comment to be sure to confirm that it does. I don't think this file is in its final form yet, sort of tried to make it as simple as possible for now.